cpp/ql/src/semmle/code/cpp/commons/Strcat.qll

```diff
@@ -8,14 +8,13 @@ import cpp
  */
 class StrcatFunction extends Function {
   StrcatFunction() {
-    exists(string name | name = getName() |
-      name = "strcat" or // strcat(dst, src)
-      name = "strncat" or // strncat(dst, src, max_amount)
-      name = "wcscat" or // wcscat(dst, src)
-      name = "_mbscat" or // _mbscat(dst, src)
-      name = "wcsncat" or // wcsncat(dst, src, max_amount)
-      name = "_mbsncat" or // _mbsncat(dst, src, max_amount)
-      name = "_mbsncat_l" // _mbsncat_l(dst, src, max_amount, locale)
-    )
+    // strcat(dst, src)
+    // strncat(dst, src, max_amount)
+    // wcscat(dst, src)
+    // _mbscat(dst, src)
+    // wcsncat(dst, src, max_amount)
+    // _mbsncat(dst, src, max_amount)
+    // _mbsncat_l(dst, src, max_amount, locale)
+    getName() = ["strcat", "strncat", "wcscat", "_mbscat", "wcsncat", "_mbsncat", "_mbsncat_l"]
```

Comment:
Better placement of these comments probably requires https://github.com/github/codeql-coreql-team/issues/548.

Reply:
I think it's still possible to place them at end-of-line if you want. The linked issue is only about a comment appearing between the opening `[` and the first set item.

Reply:
I'm not really sure what better placement would be.  Putting them all at the end of one line would be incredibly cluttered.  I could format it as one `/* */` comment with a short line of explanation at the top perhaps?

Reply:
Oh I see, @jbj is proposing something along the lines of:
```
  getName() = [
    "strcat", // strcat(dst, src)
    "strncat", // strncat(dst, src, max_amount)
    ...
  ]
```

Reply:
The above autoformats to:
```
    getName() =
      ["strcat", // strcat(dst, src)
          "strncat", // strncat(dst, src, max_amount)
          "wcscat", // wcscat(dst, src)
          ...
```
The comments are better, that first line is ugly though.

Reply:
OK, I've gone with the above (but it would be nice to have a fix for that ugly indenting at some point).